build/__nodeVersions.sh

```diff
@@ -1,15 +1,15 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-NODE_RUNTIME_BASE_TAG='20220128.5'
-YARN_VERSION='1.22.15'
-YARN_MINOR_VERSION='1.17'
+NODE_RUNTIME_BASE_TAG='20220128.5' #TODO: Depends on my PR creation date?
```

Comment:
@samruddhikhandale overall looking good! I kindly ask you test locally. More documentation on [how to test locally and deploying remotely](https://microsoft.sharepoint.com/teams/ProjectOryx/_layouts/15/Doc.aspx?sourcedoc=%7Bbbadf169-c181-4007-8661-fc5d27398240%7D&action=edit&wd=target%28Welcome.one%7C4dea46ae-e936-4365-894a-fca0f242e3e7%2FOryx%20%E2%80%93%20How%20to%27s%7Cf52fa59f-9626-4841-9c24-e3e13de1d31b%2F%29)

After you've tested locally you can run the corresponding platform pipeline, for example Oryx-BaseImages-Node pipeline and target your branch. This will generate the corresponding base tag, which is the ID of the pipeline (usually associated with the date of the invocation)

Reply:
@william-msft I don't seem to have access for the documentation. Could you help me out with that?